core/blockchain.go

```diff
@@ -555,6 +557,11 @@ func (bc *BlockChain) repair(head **types.Block) error {
 				Msg("Rewound blockchain to past state")
 			return nil
 		}
+		// Repair last commit sigs
```

Comment:
any reason of doing this?

Reply:
Because when the chain is rolled back (repair) the last commit sig needs to be updated accordingly for consensus to work.